node/forest_libp2p/src/behaviour.rs

```diff
@@ -138,42 +144,42 @@ impl NetworkBehaviourEventProcess<DiscoveryOut> for ForestBehaviour {
     }
 }
 
-impl NetworkBehaviourEventProcess<BitswapEvent> for ForestBehaviour {
+impl<P: StoreParams> NetworkBehaviourEventProcess<BitswapEvent> for ForestBehaviour<P> {
```

Comment:
Just as a heads up `NetworkBehaviourEventProcess` will go away, though that is not yet merged nor released. https://github.com/libp2p/rust-libp2p/pull/2751